src/Ampersand/PatchHelper/Helper/Magento2Instance.php

```diff
@@ -129,6 +143,111 @@ private function listXmlFiles($directories)
         sort($this->listOfXmlFiles);
     }
 
+    /**
+     * Prepare the db schema xml data so we have a map of tables to their primary definitions, and alterations
+     */
+    private function prepareDbSchemaXmlData()
+    {
+        /*
+         * Get a list of all db_schema.xml files
+         */
+        $allDbSchemaFiles = array_unique(array_filter($this->getListOfXmlFiles(), function ($potentialDbSchema) {
+            if (!str_ends_with($potentialDbSchema, '/etc/db_schema.xml')) {
```

Comment:
@convenient I believe `str_ends_with ` is a function introduced in PHP 8 https://www.php.net/manual/en/function.str-ends-with.php so this would either need refactoring or the min PHP level increased

Reply:
I've had them polyfilled in for maybe 4 years :)

Reply:
Ah all good! I didn't realise 👍🏻